dat/BIOS - Non-Merged.dat

```diff
@@ -172,19 +172,32 @@ game (
 )
 
 game (
-	name "Sega - Mega Drive - Genesis"
-	rom ( name areplay.bin size 32768 crc 95ff7c3e md5 a0028b3043f9d59ceeb03da5b073b30d sha1 1e0f246826be4ebc7b99bb3f9de7f1de347122e5 )
+	name "Sega - Game Gear"
 	rom ( name bios.gg size 1024 crc 0ebea9d4 md5 672e104c3be3a238301aceffc3b23fd6 sha1 914aa165e3d879f060be77870d345b60cfeb4ede )
-	rom ( name bios_CD_E.bin size 131072 crc 529ac15a md5 e66fa1dc5820d254611fdcdba0662372 sha1 f891e0ea651e2232af0c5c4cb46a0cae2ee8f356 )
-	rom ( name bios_CD_J.bin size 131072 crc 9d2da8f2 md5 278a9397d192149e84e820ac621a8edd sha1 4846f448160059a7da0215a5df12ca160f26dd69 )
-	rom ( name bios_CD_U.bin size 131072 crc c6d10268 md5 2efd74e3232ff260e371b99f84024f7f sha1 f4f315adcef9b8feb0364c21ab7f0eaf5457f3ed )
+)
+
+game (
+	name "Sega - Master System - Mark III"
 	rom ( name bios_E.sms size 8192 crc 0072ed54 md5 840481177270d5642a14ca71ee72844c sha1 c315672807d8ddb8d91443729405c766dd95cae7 )
 	rom ( name bios_J.sms size 8192 crc 48d44a13 md5 24a519c53f67b00640d0048ef7089105 sha1 a8c1b39a2e41137835eda6a5de6d46dd9fadbaf2 )
 	rom ( name bios_U.sms size 8192 crc 0072ed54 md5 840481177270d5642a14ca71ee72844c sha1 c315672807d8ddb8d91443729405c766dd95cae7 )
-	rom ( name ggenie.bin size 32768 crc 14dbce4a md5 e8af7fe115a75c849f6aab3701e7799b sha1 937e1878ebd104f489e6bdbc410a184f79f1144a )
-	rom ( name sk.bin size 2097152 crc 0658f691 md5 4ea493ea4e9f6c9ebfccbdb15110367e sha1 88d6499d874dcb5721ff58d76fe1b9af811192e3 )
-	rom ( name sk2chip.bin size 262144 crc 4dcfd55c md5 b4e76e416b887f4e7413ba76fa735f16 sha1 70429f1d80503a0632f603bf762fe0bbaa881d22 )
-	rom ( name rom.db size 17742 crc c94e8c8b md5 ff4a3572475236e859e3e9ac5c87d1f1 sha1 02c287d10da6de579af7a4ce73b134bbdf23c970 )
+)
+
+game (
+    name "Sega - Mega CD - Sega CD"
+    rom ( name bios_CD_E.bin size 131072 crc 529ac15a md5 e66fa1dc5820d254611fdcdba0662372 sha1 f891e0ea651e2232af0c5c4cb46a0cae2ee8f356 )
+    rom ( name bios_CD_J.bin size 131072 crc 9d2da8f2 md5 278a9397d192149e84e820ac621a8edd sha1 4846f448160059a7da0215a5df12ca160f26dd69 )
+    rom ( name bios_CD_U.bin size 131072 crc c6d10268 md5 2efd74e3232ff260e371b99f84024f7f sha1 f4f315adcef9b8feb0364c21ab7f0eaf5457f3ed )
```

Comment:
Nice work! Mind fixing the whitespace to tabs?

Reply:
![giphy](https://user-images.githubusercontent.com/25086/51090635-e0530000-174c-11e9-944a-95e31588c992.gif)